src/Parser/Parts/PartList.cs

```diff
@@ -10,6 +10,23 @@ public PartList()
         {
         }
 
+ 
+        
+        /// <summary>
+        /// Number of <see cref="TextPart"/> with type <see cref="HoleType.Numeric"/>
+        /// </summary>
+        public int NumericPartCount { get; set; }
```

Comment:
yes, that is true I guess. I checked the Serilog source code, and it was testing all numeric and minimal 1 numeric AFAIK. Had that in mind...